src/components/Home.jsx

```diff
@@ -0,0 +1,35 @@
+import { CTA } from "./CTA";
+import { Carousel } from ".//Carousel";
+import { Footer } from "./Footer";
+import { Header } from "./Header";
+import { Hero } from "./Hero";
+import { Quote } from ".//Quote";
+import { Rating } from "./Rating";
+import { useState } from "react";
+import { handleResize } from "../utils/handleResize";
+
+export const Home = () => {
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
```

Comment:
I like how you handle the resize here. I used it directly in the components. Your approach makes the code a lot cleaner.